qa/tasks/cephfs/test_volumes.py

```diff
@@ -95,6 +95,30 @@ def tearDown(self):
             self._delete_test_volume()
         super(TestVolumes, self).tearDown()
 
+    def test_volume_rm(self):
+        try:
+            self._fs_cmd("volume", "rm", self.volname)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.EPERM:
+                raise
```

Comment:
Please raise with a useful message. Something like we expected the `fs volume rm` command to fail with EPERM